packages/config/src/projects/layer2s/arbitrum.ts

```diff
@@ -295,12 +295,16 @@ export const arbitrum: Layer2 = {
         batchSize: 150,
         sinceBlock: 7654707,
         version: '3',
+        // TODO: fix this
+        isNativeBalanceSupported: false,
```

Comment:
This is dependant only on the multicall version. V3 does support native balance https://arbiscan.io/address/0xcA11bde05977b3631167028862bE2a173976CA11#code#F1#L199. What's the problem here?